.all-contributorsrc

```diff
@@ -169,6 +169,16 @@
       "contributions": [
         "doc"
       ]
+    },
+    {
+      "login": "thoriumsim",
+      "name": "thoriumsim",
+      "avatar_url": "https://avatars0.githubusercontent.com/u/48568289?v=4",
+      "profile": "https://github.com/thoriumsim",
+      "contributions": [
+        "tutorial",
```

Comment:
This should have a bit more context to it, like it's for docs as well as everything else.

```suggestion
        "docs",
        "tutorial",
```